docs/retention-policies.md

```diff
@@ -327,3 +327,73 @@ client.retentionPolicies.getAllFileVersionRetentions(options)
 		*/
 	});
 ```
+
+Get Files Under Retention For Assignment
+-------------------------------
+
+To retrieve information about files under retention, call the
+[`retentionPolicies.getFilesUnderRetentionForAssignment(assignmentID, options, callback)`](http://opensource.box.com/box-node-sdk/jsdoc/RetentionPolicies.html#getFilesUnderRetentionForAssignment) method.
+
+<!-- sample get_files_under_retention -->
+```js
+client.retentionPolicies.getFilesUnderRetentionForAssignment('12345')
+	.then(file => {
```

Comment:
looks like the response object is a collection, so you'll want to take a look at the `getAllFileVersionRetentions` example code above to shape it similarly, e.g. `.then(files => { /* files -> { entries: ...`